api/proto/teleport/legacy/types/types.proto

```diff
@@ -994,6 +994,11 @@ message AppSpecV3 {
   repeated string RequiredAppNames = 10 [(gogoproto.jsontag) = "required_app_names,omitempty"];
   // CORSPolicy defines the Cross-Origin Resource Sharing settings for the app.
   CORSPolicy CORS = 11 [(gogoproto.jsontag) = "cors,omitempty"];
+  // TCPPorts is a list of ports and port ranges that an app agent can forward connections to.
```

Comment:
```suggestion
  reserved 12;
  reserved "IdentityCenter";
  // TCPPorts is a list of ports and port ranges that an app agent can forward connections to.
```

Reply:
Is that going to conflict with #49300 which backports `IdentityCenter`? As in, wouldn't it make #49300 impossible to merge if this PR gets merged first?